cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -163,6 +175,41 @@ static Bom createBom() {
         Bom bom = new Bom();
         return bom;
     }
+
+    static void signSBOM(final String keystoreFile, final String keystorePassword, final String privateKey, final String publicKey) {
+        try {
+            // Load the keystore
+            JSONAsymKeySigner signer = new JSONAsymKeySigner(new FileInputStream(keystoreFile), keystorePassword.toCharArray(), privateKey, publicKey);
+            System.out.println("Keystore loaded successfully.");
+
+            File homeDir = new File(System.getProperty("user.home"));
+            File[] directoryListing = homeDir.listFiles();
+            if (directoryListing != null) {
+                for (File sbomFile : directoryListing) {
+                    if (sbomFile.getName().endsWith(".json")) {
+                        // read the sbom file as a json object writer
+                        JSONObjectWriter sbom = new JSONObjectWriter(new FileInputStream(sbomFile));
+                        // sign the sbom
+                        byte[] signedSbom = signer.signData(sbom.getEncoded());
+                        // Write the signed SBOM to a new file
+                        FileOutputStream signedSbomStream = new FileOutputStream("signed_" + sbomFile.getName());
```

Comment:
should we sign in the original `*sbom.json` file, instead of creating new files `signed_*sbom.json `?